### DISCUSSION TITLE
QUESTION: Docker Container for the Server itself?

### DISCUSSION THREAD

**Discussion by bmcgonag** [dyrector-io/dyrectorio#480] [Q&A]
I see that to run this I need a Go compiler, and to compile the source for now.  But could you provide a docker container to get the server / web-ui up and running, then use that to orchestrate the other servers / nodes?

I think this reduces the barrier to entry a bit.  I love the ideas and concepts shown. This looks really amazing.

Best! And thank you for making it open source!

**Comment by bmcgonag** [comment]
Maybe I found my answer. I found an docker-compose file buried in the web folder.  If this is it, then simply mention that method on the README.

**Reply by ghost** [reply]
#483 Is adding this feature.

**Comment by ghost** [comment]
We haven't got the time around for this sadly. Compose files you an find is most likely outdated since most of us are using the CLI for development and testing. Seems we have to prioritize self-hosting options.
